gradle.properties

```diff
@@ -0,0 +1 @@
+version=1.0.0-SNAPSHOT
```

Comment:
Do we not need to update this now as we have already released 1.0.0? 🤔 Thoughts?

Reply:
I thought about that too. In a normal java project we would. In that case, the version in the source would be used to generate the next tag and would then need to be incremented. However, in this situation, we're overwriting `1.0.0-SNAPSHOT` with the version that's set by creating the release. This version number is necessary to successfully build, but kind of ignored the rest of the time.

Reply:
Okay 👍